--- ./src/freeeapi/model/partnersIndexResponse.ts	2019-11-09 15:25:01.662009500 +0900
+++ ./src/freeeapi/model/partnersIndexResponse_patch.ts	2019-11-09 15:29:47.984434700 +0900
@@ -10,7 +10,8 @@
  * Do not edit the class manually.
  */
 
-import { Partner } from './partner';
+//import { Partner } from './partner';
+type Partner = any;
 
 export class PartnersIndexResponse {
     'partners': Array<Partner>;
