--- src/freeeapi/model/partnersIndexResponse.ts	2020-02-02 00:54:53.966379800 +0900
+++ src/freeeapi/model/partnersIndexResponse.ts.changed	2020-02-02 00:55:08.203344500 +0900
@@ -11,7 +11,8 @@
  */
 
 import { RequestFile } from '../api';
-import { Partner } from './partner';
+//import { Partner } from './partner';
+type Partner = any;
 
 export class PartnersIndexResponse {
     'partners': Array<Partner>;
